backend/app/models.py

```diff
@@ -43,8 +39,4 @@ class Photographer(models.Model):
     name = models.CharField(max_length=252)
     type = models.CharField(max_length=252)
     sentiment = models.CharField(max_length=252)
-    photo_objects = models.ManyToManyField(
-        'Photo',
-        blank=True,
-    )
     map_square_obj = models.ForeignKey('MapSquare', on_delete=models.SET_NULL, null=True)
```

Comment:
Before we get too deep into this project, let's rename these just `map_square` and wherever `photographer_obj` is just call it `photographer`. Typing `_obj` all the time for these is going to get tedious.